cherrypy/lib/sessions.py

```diff
@@ -94,6 +94,7 @@
 import os
 import time
 import threading
+import importlib
```

Comment:
Just add it as a marked dependency, e.g. `extras_require={':python_version=="2.6"': ['importlib']}`